packages/flutter/lib/src/cupertino/page.dart

```diff
@@ -424,19 +424,12 @@ class _CupertinoEdgeShadowDecoration extends Decoration {
   }
 
   @override
-  DiagnosticsNode toDiagnosticsNode({
-    String name,
-    DiagnosticsTreeStyle style: DiagnosticsTreeStyle.singleLine,
-  }) {
-    return new DiagnosticsNode.lazy(
-      name: name,
-      value: this,
-      style: style,
-      description: '$runtimeType',
-      fillProperties: (List<DiagnosticsNode> properties) {
-        properties.add(new DiagnosticsProperty<LinearGradient>('edgeGradient', edgeGradient));
-      },
-    );
+  String toStringShort() => '$runtimeType';
```

Comment:
all the decoration subclasses should act the same, so I think it's safe to make Decoration do whatever the BoxDecoration / FlutterLogoDecoration / _CupertinoEdgeShadowDecoration classes do (hopefully they're all consistent).